lib/console/console.rb

```diff
@@ -13,44 +12,94 @@ def greet_user
     print(messages.greet_user)
   end
 
-  def show_options
-    messages.options.each { |option| print(option) }
+  def get_players(players_factory)
+    print_options
+    prompt
+    players = players_factory.create(get_game_choice)
+    game_instructions(players.current_player_symbol)
+    players
+  end
+
+  def request_move(board = nil)
+    prompt
+    get_move
+  end
+
+  def move_summary(move, board, player, oponent)
+    clear_screen
+    print_board_with_move(move, board, player)
+    print_move_summary(move, player, oponent, board)
   end
 
-  def show_instructions(player)
+  def game_summary(board)
+    clear_screen
+    print_board(board)
+    print_terminus(board)
+  end
+
+  private
+  attr_reader :board_renderer, :messages, :io
+  def game_instructions(player)
     print(messages.instructions(player))
   end
+  
+  def print_board_with_move(move, board, player)
+    if board.valid_move?(move)
+      print_board(board.make_move(move, player))
+    else
+      print_board(board)
+    end
+  end
 
-  def show_board(board)
+  def print_board(board)
     board_string = board_renderer.render(board)
     print(board_string)
   end
 
-  def show_move_summary(move, player, oponent)
-    print(messages.player_move(move, player))
-    print(messages.player_turn(oponent))
+  def print_terminus(board)
+    print_draw    if board.full?
```

Comment:
This reads very nicely!
But couldn't it be that the board is full but there is still a winner?

Reply:
Thanks! But yes that line should be last, will add a test case for it